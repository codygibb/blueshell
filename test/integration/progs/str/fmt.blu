print "a: {}, b: {}, c: {}".fmt("A", 5, [1, 2, 3])
print "too many curlies: {} {} {}".fmt("hi")
print "not enough curlies: {} {}".fmt(1, 2, 3)
print "no curlies".fmt(5, 6, 7)
