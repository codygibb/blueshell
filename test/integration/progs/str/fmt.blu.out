a: A, b: 5, c: [1, 2, 3]
too many curlies: hi {} {}
not enough curlies: 1 2
no curlies
